vermouth/gmx/topology.py

```diff
@@ -110,7 +110,13 @@ def write_nonbond_params(system, itp_path, C6C12=False):
                     comments = ""
                 itp_file.write(f"{a1} {a2} 1 {nb1:3.8F} {nb2:3.8F} {comments}\n")
 
-def write_gmx_topology(system, top_path, itp_paths=[], C6C12=False, defines=(), header=()):
+def write_gmx_topology(system,
+                       top_path,
+                       itp_paths={"nonbond_params": "extra_nbparams.itp",
+                                  "atomtypes": "extra_atomtypes.itp"},
```

Comment:
why not?

Reply:
```py
def f(a=[]):
    a.append(42)
    return a

a1 = f()
print(a1)
a2 = f()
print(a2)
```

Because the object is made only once, when the function gets defined, which may result in "funny" side effects if it ever gets changed.